### PR TITLE
DOCKER 79 Add documentation in the DCE chart on how to generate the jwt secret

### PR DESCRIPTION
**Pull request by carminevassallo**
Our values.yml and readme.md lack consistent documentation about the jwt secret. We revise the existing guidelines and specify more where needed.

### CODE REVIEW

File: charts/sonarqube-dce/README.md [hunk 1]
```diff
@@ -16,7 +16,10 @@ Please note that this chart does NOT support SonarQube Community, Developer, and
 
 ## Installing the chart
 
-Please ensure that the value for `ApplicationNodes.jwtSecret` is set with something like `echo -n "your_secret" | openssl dgst -sha256 -hmac "your_key" -binary | base64` and persist this in your `values.yaml`.
```
Comment (leo-geoffroy-sonarsource) [inline charts/sonarqube-dce/README.md:19]: As discussed, we can put a link to the general documentation in this section

File: charts/sonarqube-dce/README.md [hunk 2]
```diff
@@ -209,7 +212,7 @@ The following table lists the configurable parameters of the Sonarqube chart and
 | `ApplicationNodes.plugins.securityContext` | Security context for the container to download plugins | see `values.yaml |
 | `ApplicationNodes.jvmOpts` | Values to add to SONARQUBE_WEB_JVM_OPTS | `""` |
 | `ApplicationNodes.jvmCeOpts` | Values to add to SONAR_CE_JAVAOPTS | `""` |
-| `ApplicationNodes.jwtSecret` | A HS256 key encoded with base64 | `""` |
+| `ApplicationNodes.jwtSecret` | A HS256 key encoded with base64 (*This value must be set before installing the chart, see [the documentation](https://docs.sonarqube.org/latest/setup/install-cluster/#header-3)*) | `""` |
```
Comment (leo-geoffroy-sonarsource) [inline charts/sonarqube-dce/README.md:215]: as discussed, update the documentation link to refer to the correct page

File: charts/sonarqube-dce/README.md (hunk 1 above)
Comment (carminevassallo) [reply charts/sonarqube-dce/README.md:19]: I just added a note including the link to the general documentation

File: charts/sonarqube-dce/README.md (hunk 2 above)
Comment (carminevassallo) [reply charts/sonarqube-dce/README.md]: yep, just updated.

**Review by leo-geoffroy-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**